basic/list2.py

```diff
@@ -13,17 +13,23 @@
 # so [1, 2, 2, 3] returns [1, 2, 3]. You may create a new list or
 # modify the passed in list.
 def remove_adjacent(nums):
-    # +++your code here+++
-    return
```

Comment:
It seems to work, but not sure why :) Pretty crazy solution. :) You are the one with R background aren't you?